### DISCUSSION TITLE
My application was rejected due to location basis despite I am in the same city as my campus

### DISCUSSION THREAD

**Discussion by Muzammilkhan785** [community/community#118266] [GitHub Education]
### Hi everyone,

Hello Everyone, I applied for Github student developer pack I had given student card as a proof but my application was rejected on location basis reasoning that I am in a different city or I am a virtual student but I am a on-campus student enrolled in Bachelors in Computer Science.Living in the same city in which my campus is. I changed my browser,cleared my cookies,I did not had any vpn turned on but still it is giving me error.If anyone can guide or help me I will be very thankful.As this student developer will be very helpful
 in my education journey and upskilling.
![error](https://github.com/community/community/assets/99830519/03e48cc5-1421-4d5c-8a63-9e9128a99860)

**Comment by tony-hj** [comment]
用通知书，好像是有要求上面要有入学日期，舍友用通知书申请成功的，我的前几天提交了，还在审批

**Reply by Muzammilkhan785** [reply]
我已经上传了我的校园学生证。上面清楚地提到了我的入学日期。但我面临的错误是位置，它说你不在校园所在的城市。但我住在那个城市。

**Comment by Jv2350** [comment]
Double-check the eligibility criteria for the GitHub student developer pack to ensure that you meet all the requirements. Sometimes there may be specific conditions that applicants need to fulfill.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.

If you want to share how you proceeded to get a successful application I'm sure others would appreciate that.
